source/clear-linux/guides/maintenance/telemetry-enable.rst

```diff
@@ -104,9 +104,13 @@ To completely remove telemetrics from your system, use the command
 Additional resources
 ********************
 
-https://clearlinux.org/features/telemetry
+*	`Telemetry feature description`_
+*	:ref:`Telemetry architecture<telemetry-about>`
+*	:ref:`Create a telemetry backend server in Clear Linux<telemetry-backend>`
+*	https://github.com/clearlinux/telemetrics-client
 
-https://github.com/clearlinux/telemetrics-client
-
-.. _`Intel's privacy policies`:
+.. _`Intel privacy policies`:
    https://www.intel.com/content/www/us/en/privacy/intel-privacy-notice.html
+
+.. _`Telemetry feature description:
+	`https://clearlinux.org/features/telemetry

```

Comment:
AR Mary fix these 2 lines so it looks like this:
.. _`Telemetry feature description`:https://clearlinux.org/features/telemetry